travis-scripts/script.sh

```diff
@@ -23,10 +23,10 @@ then
     # On osx the default gcc is actually LLVM
     export CC=gcc-7
     NO_CONFIGURE=1 ./autogen.sh
-    ./configure --enable-debug --prefix=$INSTDIR --with-init-script --with-lmdb=/usr/local/Cellar/lmdb  --with-openssl=/usr/local/opt/openssl
+    ./configure --enable-debug --prefix=$INSTDIR --bindir=$INSTDIR/var/cfengine/bin --with-init-script --with-lmdb=/usr/local/Cellar/lmdb  --with-openssl=/usr/local/opt/openssl
```

Comment:
This looks weird. Maybe all you want to do is a plain `./configure` and then `make destdir=$INSTDIR install` ?